sdk.yaml

```diff
@@ -14,148 +19,84 @@ Agent Identifier:
       Includes a product component indicating the type of runtime, as discrete from the OS type.
       This does not always include a version (e.g. for `browser` runtime emitted by `ably-js`
       and the `uwp` runtime emitted by `ably-dotnet`).
-Client Instance Creation:
-  .specification: RSC1
+Authentication:
+  .documentation:
+    - https://ably.com/docs/core-features/authentication
+  .specification: [AO1, AO2, RSC17, RSA7, RSA15, RTN2d, TO3a]
   .synopsis: |
-    An SDK announcing support for any of these construction inputs must support those
-    inputs for all client types it offers.
-  With Client Options:
-  With String:
-    .specification: RSC1a
-    .synopsis: |
-      Where that string is treated as an API key if it includes a ':' (colon) delimiter,
-      otherwise it is treated as a token string.
-Options:
-  .synopsis: |
-    `ClientOptions` that apply to both client types (REST and Realtime).
-  Agents:
-    .specification: RSC7d6
+    Mechanisms authenticating with the Ably service.
+    Encapsulated by the `AuthOptions` class.
+    Implies the existence of the `clientId` client option, the identifier of the client represented by the client instance.
```

Comment:
I'm not sure the "Authentication" feature implies the existing of a `clientId` option. For example, what if the user just supplies an API key?

Reply:
https://github.com/ably/features/pull/41/commits/07ac8a1db3e10d776fd74245f189eeb2507a9d56

Reply:
Resolving this conversation on @lmars' behalf, as I've addressed the comment and wish to close the pull request cleanly.